### ISSUE TITLE
Determine parameters around legitimacy of opting out of the pro-rata approach

### ISSUE THREAD

**Issue by hickeng** [hickeng/financial#13]
The VMW f8937 is vague:
> a shareholder ... is treated as having surrendered each share for a pro rata portion of the stock and cash received, based on the [fair market value](https://github.com/hickeng/financial/edit/main/README.md#on-fair-market-value) of such surrendered share, unless the terms of the exchange provide otherwise and are economically reasonable.

- [ ] extract the "terms of the exchange" from the M&A agreement
- [ ] read and validate TLDR for details below

## Details for historic precedent

The cost basis calculation is referenced in Code Sec. 356 (and 358):

Analysis from a NY law firm on this issue:
[Back to Basis](https://www.woodllp.com/Publications/Articles/pdf/Back_to_Basis.pdf)
[Code_Sec_356_and_358_Regulations](https://www.woodllp.com/Publications/Articles/pdf/Code_Sec_356_and_358_Regulations.pdf)
[April2006StockBasis](https://www.woodllp.com/Publications/Articles/ma/April2006StockBasis.pdf)

TLDR; "did the M&A agreement specify written down how to allocate basis or boot", and IRS won.

For VMW this seems relevant:
> There seems little chance that an acquisition agreement would try to allocate basis or boot when the target is a publicly traded company. In theory, a public target might try to draft allocations to benefit some influential insiders. But it is hard to imagine the securities lawyers signing off on such a plan.

However the f8937 for VMW/AVGO is extremely vague. [Counter-example](https://www.morganstanley.com/content/dam/msdotcom/en/about-us-ir/shareholder/Form_8937.pdf) that is very clear.
> A U.S. holder who acquired different blocks of Eaton Vance common stock at different times and at different prices generally must apply the rules described below separately to each identifiable block of shares of Eaton Vance common stock.

Even more explicit [example f8937](https://investors.delltechnologies.com/static-files/3a45254e-ad2b-4bd5-b901-a2de0a28235f)
> Section 351 Exchange and Fractional Shares
> ...
> If an EMC shareholder acquired different blocks of EMC Shares at different times or at different prices, any gain or loss should be determined separately with respect to each block of EMC Shares.
> The tax basis in each share of Class V Common Stock received in the Section 351 Exchange should reflect a blended, pro rata portion of the aggregate tax basis determined under the rules described above.

 VMW 8937 is more vague:
 "the terms of exchange provide otherwise" mean we have to go back to the M&A agreement to figure that out.

And I remember the M&A agreement is equally vague.

**Comment by hickeng** [comment]
From the merger agreement

> The stockholder election will be subject to the election of other VMware stockholders and a proration mechanism, such that the total number of shares of VMware common stock entitled to receive the cash consideration, and the total number of shares of VMware common stock entitled to receive the stock consideration, will, in each case, be equal to 50% of the aggregate number of shares of VMware common stock issued and outstanding immediately prior to the effective time of the second merger

I read this as, for any given shareholder, the ratio of shares receiving cash/stock consideration across their holding must conform to the ratio for the merger as an entirety.

**Comment by esyleung** [comment]
The wording in form 8937 is weird, and leave a lot of room for interpretation.

For instance, the other 8937 specifically state the boot + target share(s) received per source share (e.g. with wording like "each {source} share ... to {x} {target} shares, {y} cash" (e.g. AT&T and Time Warner Inc.  [https://investors.att.com/~/media/Files/A/ATT-IR/documents/stockholder-services/time-warner/Form%208937%20-%20Attachment.pdf]

In our case, it did not mention something like "per share of VMW to ...", but only "the right" to convert 1 share of VMW to $142.5, or 0.2520 shares of AVGO. (See below)

It is extremely intriguing why they worded so differently than other form 8937. 

"""
...
a) for each share of VMware Common Stock with respect to which an election to receive cash (a “Cash Election” and a “Cash Election Share”) was made, cash in an amount of $142.50 per share;
b) for each share of VMware Common Stock with respect to which an election to receive Broadcom common stock (a “Stock Election” and a “Stock Election Share”) was made, 0.2520 shares of Broadcom common stock (“Broadcom Common Stock”); and
c) for each share of VMware Common Stock for which a Cash Election or Stock Election was not validly made (collectively, “Non-Election Shares”), the right to receive such merger consideration as determined in accordance with the Merger Agreement. 
...
(I) a portion of the Stock
Election Shares of each holder was converted into the right to receive $142.50 in cash per share of VMware Common Stock, and the remaining portion of the Stock Election Shares of each holder was converted into the right to receive 0.2520 of a share of Broadcom Common Stock per share of VMware Common Stock, and 
(II) all Cash Election Shares and Non-Election Shares were converted into the right to receive $142.50 in cash per share of VMware Common stock. 
"""

**Comment by cwei44** [comment]
if we set k26 of first sheet to "Manual per-lot ratio", and on RSU/ESPP sheets set each of the lots individually as cash or shares, and the accumulation of shares(converted) matches Total_Share * 0.520866, I can see the tax was reduced significantly, but the question is, would this be a legit approach to fill out our tax-return?

**Comment by hickeng** [comment]
@cwei44 answering that is the purpose of this issue.

I've spoken to several people trained as CPAs (but engaged as one by me) and they've said that lot selection is a valid method. However they were unable to comment on whether there are any restrictions on who/how/when it's valid.

Finding a CPA who:
* is knowledgeable about the specifics of cash-to-boot and lot selection, and
* has capacity to take on a client, and
* is willing to answer questions like this generally rather than just for a specific individual while filing

is one of my primary goals. It's proving challenging.

**Comment by hickeng** [comment]
Added [this analysis of merger sections 356 and 358](https://www.woodllp.com/Publications/Articles/pdf/Code_Sec_356_and_358_Regulations.pdf) to the repo. Very informative, but still need assistance in interpretation.

**Comment by hickeng** [comment]
There's [this section of IRS code](https://www.law.cornell.edu/cfr/text/26/1.358-2#:~:text=(vii)%20If,purchased%20or%20acquired.) ( §1.358-2(a)(2)(vii) ) which notes the basis assignment can be done at any point before it becomes tax-relevant.

Not sure what that means for the fractional share and the lot it comes from - sale/disposition is definitely tax relevant. Not sure what else is.

**Comment by cwei44** [comment]
I recently transferred my shared from etrade to morgan stanley, and found all the tax lots are still there, i think this can indicate that all the lots were applied 48/52 as cash/convert ratio. so meaning i have to use "balanced".

**Comment by hickeng** [comment]
@cwei44 Lots are a mechanism for grouping fungible electronic shares. I would not have expected them to be lost in the transfer because the acquisition date, basis value, and whether it's covered/non-covered need to be preserved.

That's distinct from saying, "this is how I'm applying the transaction across my lots". The purpose of this issue is to track whether choosing an assignment other than what eTrade decided is defensible.

**Comment by 0guanhua0** [comment]
https://www.irs.gov/pub/irs-pdf/p550.pdf

65-66

Cost Basis
You can figure your gain or loss using a cost basis only if
you did not previously use an average basis for a sale, ex-
change, or redemption of other shares in the same mutual
fund.
To figure cost basis, you can choose one of the follow-
ing methods.
• Specific share identification.
• First-in first-out (FIFO).

Average Basis
